free-science-books.md

```diff
@@ -1,14 +1,31 @@
 # Subject areas (In order of LIBRARY OF CONGRESS CLASSIFICATION)
 
 ## Science (General)   (Q)
+* [Fundamentals of Piano Practice](http://www.pianopractice.org/)
+
 ## Mathematics (QA)
+
 ## Astronomy   (QB)
+
 ## Physics (QC)
+* [The Feynman Lectures on Physics](http://feynmanlectures.caltech.edu/)
+* [Quantum Diaries](http://www.quantumdiaries.org/)
+
```

Comment:
But a great learning resource. I envisioned the whole more like an awesome list of learning resources concerning sciences. But ok only books.